components/datetime/src/fields/symbols.rs

```diff
@@ -697,3 +731,55 @@ impl LengthType for TimeZone {
         }
     }
 }
+
+/// A second field with fractional digits.
+#[derive(
+    Debug, Eq, PartialEq, Ord, PartialOrd, Clone, Copy, yoke::Yokeable, zerofrom::ZeroFrom,
+)]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, databake::Bake),
+    databake(path = icu_datetime::fields),
+)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+#[allow(clippy::enum_variant_names)]
+#[repr(u8)]
+#[zerovec::make_ule(DecimalSecondULE)]
+#[zerovec::derive(Debug)]
+#[allow(clippy::exhaustive_enums)] // used in data struct
+pub enum DecimalSecond {
```

Comment:
`question, n-b`: what's the advantage of this over `DecimalSecond(u8)`?

Reply:
- Can exhaustively match
- More niches
- Encodes the invariants into the type itself